Dockerfile

```diff
@@ -19,4 +19,5 @@ RUN cd /usr/bin && \
     ln -s control-plane-operator token-minter
 
 ENTRYPOINT /usr/bin/hypershift
-LABEL io.openshift.hypershift.control-plane-operator-subcommands=true
\ No newline at end of file
+LABEL io.openshift.hypershift.control-plane-operator-subcommands=true
+LABEL io.openshift.hypershift.ignition-server-healthz-handler=true
```

Comment:
this being true does not guarantees `ignition-server/cmd/start.go` has the change introduced by this PR. Shouldn't this be in Dockerfile.control-plane instead?

Reply:
Also once we move ign-server reconciliation down to CPO we'll need to have `io.openshift.hypershift.ignition-server-reconciled-by-cpo` then we can introduce the handler change transparently without introducing an additional label.

Reply:
true, also needed in the other Dockerfile